.goreleaser.yml

```diff
@@ -21,22 +53,22 @@ env:
   - CGO_ENABLED=1
   - GO111MODULE=on
 archives:
-- replacements:
-    darwin: macOS
-    linux: Linux
-    windows: Windows
-    386: 32-bit
-    amd64: 64-bit
-  format_overrides:
-    - goos: windows
-      format: zip
+  - replacements:
```

Comment:
Does this work with this indentation?

Reply:
yes, i didn't format this manually btw, i use prettier plugin in vscode

Reply:
Okay, let's merge this and try to do a release!